src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -20,6 +21,8 @@ partial class ApplicationEngine
         public static readonly InteropDescriptor System_Contract_Call = Register("System.Contract.Call", nameof(CallContract), 0_01000000, TriggerType.System | TriggerType.Application, CallFlags.AllowCall, false);
         public static readonly InteropDescriptor System_Contract_CallEx = Register("System.Contract.CallEx", nameof(CallContractEx), 0_01000000, TriggerType.System | TriggerType.Application, CallFlags.AllowCall, false);
         public static readonly InteropDescriptor System_Contract_IsStandard = Register("System.Contract.IsStandard", nameof(IsStandardContract), 0_00030000, TriggerType.All, CallFlags.None, true);
+        public static readonly InteropDescriptor System_Contract_IsDeployed = Register("System.Contract.IsDeployed", nameof(IsDeployedContract), 0_00030000, TriggerType.All, CallFlags.None, true);
+        public static readonly InteropDescriptor System_Contract_IsPayable = Register("System.Contract.IsPayable", nameof(IsPayableContract), 0_00030000, TriggerType.All, CallFlags.None, true);
```

Comment:
Or just create a `System.Contract.Get`, which can return the metadata of the contract? Then we don't need so many `System.Contract.IsXXX` syscalls.

Reply:
I thought this at the beginning, i can change it if it's more clear

Reply:
`GetInfo` or jus `Get`?

Reply:
But we also have this one https://github.com/neo-project/neo/pull/1729

Reply:
In fact, we have this `System.Blockchain.GetContract`

Reply:
`IsDeployed` and `IsPayable` are not needed.

Reply:
Agree, we can use `System.Blockchain.GetContract` for this